Lib/inspect.py

```diff
@@ -899,6 +899,7 @@ def __init__(self):
         self.indecorator = False
         self.decoratorhasargs = False
         self.last = 1
+        self._decorator_parens = []
```

Comment:
I wonder if this should be `decoratorparens` to match the other attributes

Reply:
Makes sense, will do that as well.